web/src/hotspots.js

```diff
@@ -229,7 +231,7 @@ function insert_hotspot_into_DOM(hotspot) {
             window.addEventListener(
                 event_name,
                 _.debounce(() => {
-                    if (place_icon(hotspot)) {
+                    if (hotspot.has_trigger || place_icon(hotspot)) {
```

Comment:
Can you explain why we're adding `hotspot.has_trigger` to these conditionals? I don't understand the logic.